cores/esp8266/Schedule.cpp

```diff
@@ -122,6 +121,13 @@ bool schedule_recurrent_function_us(const std::function<bool(void)>& fn,
 
     esp8266::InterruptLock lockAllInterruptsInThisScope;
 
+    // prevent new item overwriting an already expired rTarget.
+    const int32_t remaining = rTarget - micros();
+    if (!rFirst || (remaining > 0 && static_cast<uint32_t>(remaining) > item->callNow.remaining()))
+    {
+        rTarget = micros() + item->callNow.remaining();
```

Comment:
`item->callNow.remaining()` is called twice.
Should it be cached ?

Reply:
Done.